### PR TITLE
Added Helper Annotation

### PR DESCRIPTION
**Pull request by therealssj**
Issue https://github.com/d8-contrib-modules/tfa/issues/28

### CODE REVIEW

File: src/Plugin/TfaBasePlugin.php [hunk 1]
```diff
@@ -61,7 +61,8 @@
    *       Plugin-specific context for use during TfaSetup.
    *
    */
-  public function __construct(array $context) {
+  public function __construct(array $context, array $configuration, $plugin_id, $plugin_definition) {
```
Comment (nerdstein) [inline src/Plugin/TfaBasePlugin.php:64]: How does this link in with the annotation?

Comment (nerdstein) [inline src/Plugin/TfaBasePlugin.php:64]: This appears to be the base plugin and not the setup plugin

Comment (therealssj) [inline src/Plugin/TfaBasePlugin.php:64]: The pluginDefinitions are setup in PluginBase class which this class extends.
All the plugins extend this class.
This is somewhat same as PasswordConstraintBase class.

Comment (nerdstein) [inline src/Plugin/TfaBasePlugin.php:64]: OK, merging.

**Pull request merged**